docs/src/pages/en/media-clip-selector.md

```diff
@@ -0,0 +1,42 @@
+---
+title: <media-clip-selector>
+description: Media Clip Selector
+layout: ../../layouts/MainLayout.astro
+source: https://github.com/muxinc/media-chrome/tree/main/src/js/extras/media-clip-selector/index.js
+---
+
+A draggle UI on the time range for creating media clips.
+
+<img src="/assets/clip-selector.png" alt="Media Clip Selector" />
+
+**Note:** `<media-clip-selector>` is an extra, which means it is not imported with the default bundle, it needs to be imported as-needed from the dist directory, for example:
```

Comment:
**Note:** `<media-clip-selector>` is an extra, which means it is not imported with the default bundle. To use it, you will need to import it from the `dist` directory manually. For example: